examples/custom-databases/mongoose.js

```diff
@@ -0,0 +1,111 @@
+const Discord = require('discord.js'),
+    client = new Discord.Client(),
+    settings = {
+        prefix: 'g!',
+        token: 'Your Discord Token',
+    };
+
+const mongoose = require('mongoose');
+
+// Connect to database
+mongoose.connect('mongodb://localhost/giveaways', { useFindAndModify: false });
+const db = mongoose.connection;
+
+// Check the connection
+db.on('error', console.error.bind(console, 'Connection error:'));
+db.once('open', () => {
+    console.log('Connected to MongoDB.');
+});
+
+// Create the schema for giveaways
+const giveawaySchema = new mongoose.Schema({
+    messageID: String,
+    channelID: String,
+    guildID: String,
+    startAt: Number,
+    endAt: Number,
+    ended: Boolean,
+    winnerCount: String,
```

Comment:
I think this should be a number
```suggestion
    winnerCount: Number,
```

Reply:
It looks string in the json

Reply:
It should not be a string. It could be a string if you use:
```js
.start(message.channel, { winnerCount: '4' });
// instead of 
.start(message.channel, { winnerCount: 4 });
```
but you should not do that

Reply:
The mongoose schema should match this interface:
https://github.com/Androz2091/discord-giveaways/blob/master/typings/index.d.ts#L142
(some are missing such as extraData or reaction)

Reply:
I totally forgot the extraData, that's the new feature right?